install/resources/upgrade43.php

```diff
@@ -35,7 +35,13 @@ function upgrade43_dbchanges()
 		$db->update_query('settings', array('value' => 1), "name='captchaimage'"); // Reset CAPTCHA to MyBB Default
 		$db->update_query('settings', "value=''", 'name IN (\'captchapublickey\', \'captchaprivatekey\''); // Clean out stored credential keys
 	}
-
+	
+	if($db->field_exists('users', 'aim'))
```

Comment:
The order of arguments for [`DB_Base::field_exists()`](https://github.com/mybb/mybb/blob/mybb_1815/inc/db_base.php#L156) and [`DB_Base::drop_column()`](https://github.com/mybb/mybb/blob/mybb_1815/inc/db_base.php#L362) is inverted here

Reply:
Oops! Corrected.
https://github.com/mybb/mybb/pull/3220